charts/portworx/templates/portworx-lighthouse.yaml

```diff
@@ -21,20 +21,20 @@ rules:
     resources: ["nodes", "services"]
     verbs: ["get", "list", "watch"]
   - apiGroups: ["stork.libopenstorage.org"]
-    resources: ["clusterpairs"]
-    verbs: ["get", "list", "create", "update", "delete"]  
+    resources: ["clusterpairs","migrations"]
```

Comment:
stork at install.portworx.com also has `groupvolumesnapshots. Can you also add that here?

Reply:
yessir

Reply:
Any other sections that might have drastically changed?
I'll update them but feature owners should maintain their pieces IMO.